go/obscuronode/host/host.go

```diff
@@ -538,30 +466,114 @@ func (a *Node) monitorBlocks() {
 
 	// only process blocks if the node is running
 	for atomic.LoadInt32(a.stopNodeInterrupt) == 0 {
-		latestBlkHeader := <-listener
+		blkHeader := <-listener
+
 		// don't process blocks if the node is stopping
 		if atomic.LoadInt32(a.stopNodeInterrupt) == 1 {
 			return
 		}
-		block, err := a.ethClient.BlockByHash(latestBlkHeader.Hash())
+
+		// ignore blocks if bootstrapping is happening
+		if atomic.LoadInt32(a.ignoreBootstrapping) == 0 {
```

Comment:
I'd be tempted to make a node method like `a.bootstrappingInProgress() bool` or something to hide the implementation details of this check away, then you wouldn't really need the comment because it's self-documenting

Reply:
The updated var name makes it clearer I think, would avoid creating methods if they are not reused.